halibot/halibot.py

```diff
@@ -53,7 +53,8 @@ def start(self, block=True):
 		self.running = True
 
 		if self.use_config:
-			self._load_config()
+			if not self.config:
+				self._load_config(configfile=self.configfile)
```

Comment:
Forgot to mention this in the description, but I have it here skip loading the config if it was already defined. We may want to change this behavior, but I figure if someone is calling `.start()`, they can also call `._load_config` as well.

Reply:
Follow up `TODO` item: we should really consider `config.json` validation.